code/frameworks/angular/src/client/docs/sourceDecorator.ts

```diff
@@ -38,8 +38,8 @@ export const sourceDecorator = (
 
   useEffect(() => {
     if (toEmit) {
-      const { id, args } = context;
-      channel.emit(SNIPPET_RENDERED, { id, args, source: toEmit, format: 'angular' });
+      const { id, unmappedArgs } = context;
+      channel.emit(SNIPPET_RENDERED, { id, args: unmappedArgs, source: toEmit, format: 'angular' });
```

Comment:
Do we want to call this `unmappedArgs` just to be really clear about what's going on here? It's a breaking change, but I don't really consider this to be a public API so I think we can get away with it.

Reply:
Yeah, I wasn't sure. I tend to agree with you, I don't know about the "breaking" change though.

Reply:
@shilman I sort of have second thoughts about this. All the other events `ARGS_UPDATED`, `STORY_PREPARED`, etc, emit the unmapped args on a key called `args`. So it would be consistent to leave it.